bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Higgs/dalitz_no_mll_cut/WH_012j_NLO_HtoElElGamma/WH_012j_NLO_HtoElElGamma_extramodels.dat

```diff
@@ -0,0 +1,4 @@
+HC_NLO_X0_UFO-v1.3.zip
+
+HC_NLO_X0_UFO-lepton_masses_no_lepton_yukawas-v1.3.tar.gz
```

Comment:
What is the reason to have both here?

Reply:
import model HC_NLO_X0_UFO-lepton_masses_no_lepton_yukawas --bypass_check
 
 It imports a different model in MadSpin for decays so maybe this is why

Reply:
My point is: The zipped model appears in the extramodel file but no where else. Why?